src/pwa/stores/lang.js

```diff
@@ -0,0 +1,22 @@
+import { types, getRoot } from 'mobx-state-tree';
+import * as languages from './languages';
+
+export default types
+  .model('Lang')
+  .props({
+    default: types.frozen(languages.en),
```

Comment:
Just curious: why is `default` a prop and `current` a view? What is different between them?

Reply:
Good question, I don't know either... 🤔
Actually, this has happened for copying code. I guess I could get rid of `default` just using `languages.en` [here](https://github.com/frontity/wp-org-comments/pull/1/files/38ca5bb7e46f00c949ca1ddcbe24d79b7ab7ee08#diff-e8fd0a5b429059b17ecd35395019eaa0R20) instead of `self.default`.